app-backend/backsplash-core/src/main/scala/com/rasterfoundry/backsplash/MosaicImplicits.scala

```diff
@@ -272,7 +242,7 @@ class MosaicImplicits[HistStore: HistogramStore, RendStore: RenderableStore](
         }
     }
 
-  val paintedMosaicTmsReification: TmsReification[BacksplashMosaic] =
+  def paintedMosaicTmsReification: TmsReification[BacksplashMosaic] =
```

Comment:
?

Reply:
oh, at one point in the refactor I think this was taking an argument, but that got moved out - I should switch this back